server/routers/jobs_router.py

```
@@ -270,249 +433,155 @@ def load_fiche_metier():
+            "Authorization": f"Bearer {token}",
+            "Accept": "application/json"
+        }
+
+        fiche_metier = []
+        
+        champs = "?champs=accesemploi,appellations(code,classification,libelle),centresinteretslies(centreinteret(libelle,code,definition)),code,libelle,competencesmobilisees(libelle,code),contextestravail(libelle,code,categorie),definition,domaineprofessionnel(libelle,code,granddomaine(libelle,code)),metiersenproximite(libelle,code),secteursactiviteslies(secteuractivite(libelle,code,secteuractivite(libelle,code,definition),definition)),themes(libelle,code),emploicadre,emploireglemente,transitiondemographique,transitionecologique,transitionnumerique"
+
+        for code in tqdm(code_metier):
+            try:
+                while True:
+                    resp = requests.get(FT_API_URL_METIER + f"/{code.get("code")}" + champs, headers=headers)
```

Ruff (0.14.13)

Probable use of `requests` call without timeout

(S113)